python/pyspark/sql/functions.py

```diff
@@ -975,7 +975,7 @@ def date_format(date, format):
     A pattern could be for instance `dd.MM.yyyy` and could return a string like '18.03.1993'. All
     pattern letters of `datetime pattern`_. can be used.
 
-    .. _datetime pattern: https://spark.apache.org/docs/latest/sql-ref-datetime-pattern.html
+    .. _datetime pattern: /docs/latest/sql-ref-datetime-pattern.html
```

Comment:
@yaooqinn you can just use the `http://...` links.

Seems the relative links work in your local build but doesn't work when it's released because the documentation root is rebased.

Reply:
thanks,fixed